confidant_client/__init__.py

```diff
@@ -430,9 +440,11 @@ def _get_keys_and_encrypted_pairs(
                 blind_key,
                 _kms
             )
-            data_keys[region] = base64.b64encode(
-                ensure_bytes(data_key['ciphertext'])
-            ).decode('ascii')
+            data_keys[region] = ensure_text(
+                base64.b64encode(
+                    ensure_bytes(data_key['ciphertext'])
+                )
+            )
```

Comment:
`b64encode` is *always* going to return `bytes` so it should use `.decode('ASCII')` and not `ensure_text` which guesses.

Reply:
I think the same is true for `f.encrypt()` below, but I'm not familiar with that API